SU2_CFD/src/solvers/CHeatSolver.cpp

```diff
@@ -377,8 +377,8 @@ void CHeatSolver::Upwind_Residual(CGeometry *geometry, CSolver **solver_containe
       V_i = solver_container[FLOW_SOL]->GetNodes()->GetPrimitive(iPoint);
       V_j = solver_container[FLOW_SOL]->GetNodes()->GetPrimitive(jPoint);
 
-      Temp_i_Grad = nodes->GetGradient(iPoint);
-      Temp_j_Grad = nodes->GetGradient(jPoint);
+      const auto Temp_i_Grad = nodes->GetGradient(iPoint);
+      const auto Temp_j_Grad = nodes->GetGradient(jPoint);
```

Comment:
now auto evaluates to the MatrixView type, right (that returns const values on its own)?

Reply:
Correct. Before it was a const pointer that could return non const